app/javascript/packs/stylesheets/layout.scss

```diff
@@ -18,3 +18,7 @@
   font-style: italic;
   float: right;
 }
+
+.custom-nav-link {
+  padding-bottom: 2px;
```

Comment:
is this point of this to make these look more aligned with the "Sara Alert v1.16.0" text? I think this is causing them to not be aligned with the navbar components on the right

![Screen Shot 2020-11-04 at 1 31 36 PM](https://user-images.githubusercontent.com/9956561/98154016-14f3f680-1ea2-11eb-81d2-6728e044911c.png)

Reply:
Yeah that's what it's aligning with, and you're right it does make the items on the right mis-aligned. This was an original request when adding those tabs in (which I get because it looks off when not aligned with the Sara Alert text too). I will double check with PO team.

Reply:
Decision was to align everything.